### PR TITLE
Add possibility to inehrit master template including styles

### PR DESCRIPTION
**Pull request by nsams**
- add renderer.getComponentMasterTemplate for inheriting in twig
- add masterExtends setting for inheriting css

### CODE REVIEW

File: Kwf/Component/Renderer/Twig/TemplateLocator.php [hunk 1]
```diff
@@ -1,49 +1,64 @@
...
-        $pos = strpos($componentClass, '_');
-        $ns1 = substr($componentClass, 0, $pos+1);
-
-        $pos = strpos($componentClass, '_', $pos+1);
-        if ($pos !== false) {
-            $ns2 = substr($componentClass, 0, $pos+1);
+        $file = null;
+        if ($type == 'Master'
+            && Kwc_Abstract::hasSetting($componentClass, 'masterTemplate')
+            && Kwc_Abstract::getSetting($componentClass, 'masterTemplate')
```
Comment (kaufmo) [inline Kwf/Component/Renderer/Twig/TemplateLocator.php]: hasSetting is enough for check?

Comment (nsams) [inline Kwf/Component/Renderer/Twig/TemplateLocator.php]: not if it is set to false

Comment (kaufmo) [inline Kwf/Component/Renderer/Twig/TemplateLocator.php]: ah ok, then :+1:

**Comment by fraxachun** [comment]
:+1:

**Pull request merged**